polyfill/lib/ecmascript.mjs

```diff
@@ -1354,7 +1353,18 @@ function bisect(getState, left, right, lstate = getState(left), rstate = getStat
   }
   return right;
 }
-
+function unique(arr) {
```

Comment:
This whole function could be `return [...new Set(arr)]`

Reply:
Unfortunately, that doesn't seem to work with the wrappers from the big-integer library.

Reply:
ah, because they don't use `===`, gotcha.